nixos/modules/services/web-apps/miniflux.nix

```diff
@@ -0,0 +1,131 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+let
+  cfg = config.services.miniflux;
+
+  dbUser = "miniflux";
+  dbPassword = "miniflux";
+  dbHost = "localhost";
+  dbName = "miniflux";
+in
+
+{
+  options = {
+    services.miniflux = {
+      enable = mkEnableOption "miniflux";
+
+      user = mkOption {
+        type = types.str;
+        default = "miniflux";
+        description = ''
+          User account under which miniflux runs.
+        '';
+      };
+
+      group = mkOption {
+        type = types.str;
+        default = "miniflux";
+        description = ''
+          User group under which miniflux runs.
+        '';
+      };
+
+      listenAddress = mkOption {
+        type = types.str;
+        default = "localhost";
+        description = ''
+          The host the daemon will listen on.
+        '';
+      };
+
+      listenPort = mkOption  {
+        type = types.int;
+        default = 8080;
+        description = ''
+          The port the daemon will listen on.
+        '';
+      };
+
+      adminCredentialsFile = mkOption  {
+        type = types.str;
+        default = "";
+        description = ''
+          File containing the ADMIN_USERNAME and ADMIN_PASSWORD (length >= 6),
+          in the format of an EnvironmentFile as described by systemd.exec(5).
+        '';
+        example = "/etc/nixos/miniflux-admin-credentials";
+      };
+    };
+  };
+
+  config = mkIf cfg.enable {
+    services.postgresql.enable = true;
+
+    users = {
+      users.miniflux = {
+        name = cfg.user;
+        group = cfg.group;
+        description = "miniflux service user";
+        isSystemUser = true;
+        createHome = false;
+      };
+      groups.miniflux = {
+        name = cfg.group;
+      };
+    };
+
+    systemd.tmpfiles.rules = [
+      "D '/run/miniflux' 700 ${cfg.user} ${cfg.group} 0"
```

Comment:
Unless you have special reason not to in this case you should use `RuntimeDirectory = "miniflux";` and `RuntimeDirectoryMode = "0700";` instead.

Reply:
It's more explicit and has been changed to it.